### PR TITLE
Added troubleshooting point

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added troubleshooting point

### CODE REVIEW

File: docs/Troubleshooting/Troubleshooting.md [hunk 1]
```diff
@@ -65,6 +65,28 @@ When running Pantheon from the Docker image a number of [command line options ar
 Not including a sufficient milestone in the genesis file can lead to unexpected and inconsistent behaviour without
 specific errors. Ensure a sufficient milestone is included in the genesis file (for example, `constantinoplefixblock`). 
 
+## Illegal Reflective Access Error on Startup
+
+The following error message may be displayed on startup and does not affect the operation of Pantheon:
```
Comment (shemnon) [inline docs/Troubleshooting/Troubleshooting.md]: I would add in that this occurs with Java versions 9 or later, and that this error will not be seen with Java 8.

File: docs/Troubleshooting/Troubleshooting.md [hunk 2]
```diff
@@ -65,6 +65,28 @@ When running Pantheon from the Docker image a number of [command line options ar
...
+```
+
+To stop the message being displayed, add the following to the environment: 
+
+```bash tab="Linux / Mac OS"
+export PANTHEON_OPTS="--add-opens java.base/sun.security.provider=ALL-UNNAMED"
+```
+    
+```bash tab="Windows"
+set PANTHEON_OPTS="--add-opens java.base\sun.security.provider=ALL-UNNAMED"
```
Comment (shemnon) [inline docs/Troubleshooting/Troubleshooting.md]: It's a forward slash still.  IT's not a path separator but a Java separator.

File: docs/Troubleshooting/Troubleshooting.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Troubleshooting/Troubleshooting.md]: Fixed

File: docs/Troubleshooting/Troubleshooting.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Troubleshooting/Troubleshooting.md]: Fixed

**Pull request merged**